.github/workflows/connector_integration_test_single_dagger.yml

```diff
@@ -9,11 +9,6 @@ on:
   pull_request:
     paths:
       - "airbyte-integrations/connectors/**"
-  push:
```

Comment:
@cpdeethree I disabled runs on master because I want these pipelines to only be executed in PR contexts. We'll create a different one for nightly builds (still calling the dagger pipeline but with different arguments)